nf-test.config

```diff
@@ -1,6 +1,16 @@
 config {
+    // location for all nf-tests
     testsDir "tests"
+
+    // nf-test directory including temporary files for each test
     workDir ".nf-test"
+
+    // location of library folder that is added automatically to the classpath
+    libDir "tests/pipeline/lib/"
+
+    // location of an optional nextflow.config file specific for executing tests
     configFile "nextflow.config"
+
+    // run all test with the defined docker profile from the main nextflow.config
     profile "test,docker"
```

Comment:
Still think this shouldn't be set to docker.

Reply:
agreed. for now we let this pass. I am going to raise an issue on profile addition rather than override on nf-test